recipes/range-v3/all/conanfile.py

```diff
@@ -8,12 +10,35 @@ class Rangev3Conan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     description = "Experimental range library for C++11/14/17"
     topics = ("range", "range-library", "proposal", "iterator")
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     @property
     def _source_subfolder(self):
         return os.path.join(self.source_folder, "source_subfolder")
 
+    def _validate_compiler_settings(self):
```

Comment:
Use https://github.com/conan-io/conan-center-index/blob/52cb2a3d4aab8a3075d4b1b90bd353e7309cef25/recipes/taocpp-json/all/conanfile.py#L28 approach instead.

cppstd should be check if cppstd is configured. By default it's not.

Reply:
I am doing this check for windows since it is required, however the other compilers do not follow the same rule

Reply:
check_min_cppstd should be called only if settings.compiler.cppstd is defined.

Reply:
Thank you for repeating, now I understand